CONTRIBUTING.md

```diff
@@ -24,4 +24,109 @@ File your feature request through GitHub Issues using the [Feature Request](http
 In the process of shipping features quickly, we may forget to keep our docs up to date. You can help by suggesting improvements to our documentation using the [Documentation Improvement](https://github.com/keploy/docs/issues) template!
 
 #### ⚙️ Close a Bug / Feature issue
-We welcome contributions that help make keploy bug free & improve the experience of our users. You can also find issues tagged [Good First Issues](https://github.com/keploy/keploy/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
\ No newline at end of file
+We welcome contributions that help make keploy bug free & improve the experience of our users. You can also find issues tagged [Good First Issues](https://github.com/keploy/keploy/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
+
+# How to Contribute
+
+## Prerequisites
+
+Make sure that the following prequisites are installed in your Operating System before you start contributing to the project - : 
+
+- [Go](https://go.dev/)
+
+To verify run :
+
+```
+go version
+```
+
+## Set up your Local Development Environment
+
+Follow the following instructions to start contributing - 
+
+1 . Fork [this](https://github.com/keploy/go-sdk.git)
+
+2 . Clone the copy of your forked project
+
+```
+git clone https://github.com/<your-github-username>/go-sdk.git
+```
+
+3 . Navigate to the project directory
+
+```
+cd go-sdk
+```
+4 . Add a remote reference (upstream) to the original repository.
+
+```
+git remote add upstream https://github.com/keploy/go-sdk.git
+```
+
+5 . Always take a pull from the upstream repository to your master branch to keep it updated with the main project.
+
+```
+git pull upstream main
+```
+
+6 . Configure the pre-commit hook by running the following path.
+
+```
+git config core.hooksPath .githooks
```

Comment:
We also need to give the scripts inside `.githooks` permission for execution. So this line can become `git config core.hooksPath .githooks && chmod +x .githooks/*`

Reply:
okay will do the needfull!